omega/omega-transaction/src/main/java/org/apache/servicecomb/saga/omega/transaction/TransactionAspect.java

```diff
@@ -40,26 +40,29 @@
   private static final Logger LOG = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
   private final OmegaContext context;
+  private final EventAwareInterceptor interceptor;
   private final ScheduledExecutorService executor = Executors.newSingleThreadScheduledExecutor();
-  private final CompensableInterceptor interceptor;
 
-  public TransactionAspect(MessageSender sender, OmegaContext context) {
+  public TransactionAspect(MessageSender messageSender, OmegaContext context) {
     this.context = context;
-    this.interceptor = new CompensableInterceptor(context, sender);
+    this.interceptor = new CompensableInterceptor(context,messageSender);
   }
 
   @Around("execution(@org.apache.servicecomb.saga.omega.transaction.annotations.Compensable * *(..)) && @annotation(compensable)")
   Object advise(ProceedingJoinPoint joinPoint, Compensable compensable) throws Throwable {
     Method method = ((MethodSignature) joinPoint.getSignature()).getMethod();
     LOG.debug("Intercepting compensable method {} with context {}", method.toString(), context);
 
-    String signature = compensationMethodSignature(joinPoint, compensable, method);
+    Object[] args = joinPoint.getArgs();
```

Comment:
retry probably needs to run synchronously instead of async. when alpha receives TxAbortedEvent, it needs to ask Omega to retry immediately. Or the transaction will fail on user service side.